lib/hardware_services/network_drivers/hilapi.py

```diff
@@ -0,0 +1,107 @@
+# this is a library to conveniently do REST calls to HIL server
+
+import requests
+import sys
+import yaml
+
+def error_check(response):
```

Comment:
This file is an encapsulation of how to talk to this HIL API, which is great.  What code uses this?

I was under the impression that this was part of the HIL Network Driver, but if this is used by both the HIL Inventory and Network drivers, then having a separate module for it is okay.

Still, as the URLs are about objects, perhaps making this into an object based class would be worth the time.  At least consider what it would take to do that to see if it is worth it.